central/config/service/service.go

```diff
@@ -125,3 +128,45 @@ func (s *serviceImpl) PutConfig(ctx context.Context, req *v1.PutConfigRequest) (
 	}
 	return req.GetConfig(), nil
 }
+
+// GetVulnerabilityDeferralConfig returns Central's vulnerability deferral configuration.
+func (s *serviceImpl) GetVulnerabilityDeferralConfig(ctx context.Context, _ *v1.Empty) (*v1.GetVulnerabilityDeferralConfigResponse, error) {
+	if !env.UnifiedCVEDeferral.BooleanSetting() {
+		return nil, errors.Errorf("Cannot fulfill request. Environment variable %s=false", env.UnifiedCVEDeferral.EnvVar())
+	}
+	config, err := s.datastore.GetConfig(ctx)
+	if err != nil {
+		return nil, err
+	}
+	return &v1.GetVulnerabilityDeferralConfigResponse{
+		Config: VulnerabilityDeferralConfigStorageToV1(config.GetPrivateConfig().GetVulnerabilityDeferralConfig()),
+	}, nil
+}
+
+// UpdateVulnerabilityDeferralConfig updates Central's vulnerability deferral configuration.
+func (s *serviceImpl) UpdateVulnerabilityDeferralConfig(ctx context.Context, req *v1.UpdateVulnerabilityDeferralConfigRequest) (*v1.UpdateVulnerabilityDeferralConfigResponse, error) {
+	if !env.UnifiedCVEDeferral.BooleanSetting() {
+		return nil, errors.Errorf("Cannot fulfill request. Environment variable %s=false", env.UnifiedCVEDeferral.EnvVar())
+	}
+	if req.GetConfig() == nil {
+		return nil, errors.Wrap(errox.InvalidArgs, "vulnerability deferral config must be specified")
+	}
+	config, err := s.datastore.GetConfig(ctx)
+	if err != nil {
+		return nil, err
+	}
+	if config == nil {
+		config = &storage.Config{}
```

Comment:
For my education, what does an update to delete the private config do to the processing?  Does that essentially mean retention periods of infinity for example?

Reply:
The zero values mean different for different retentions. For example, for images, it means no retention at all i.e. collect during the next garbage collection cycle, whereas, for decommissioned clusters, the zero values mean do not garbage collect.